Src/signal.c

```diff
@@ -160,7 +160,13 @@ void transfercomplete()
                     buffersize = 2;
                     computeServoInput();
                 }
+#ifdef MCU_CH32V203
```

Comment:
why not make these changes part of receiveDShotDma() ?

Reply:
It can indeed be moved to receiveDShotDma()